numba/pycc/cc.py

```diff
@@ -1,6 +1,9 @@
-from distutils import dir_util, log
-from distutils.command import build_ext
-from distutils.extension import Extension
+from setuptools import distutils as dutils
+dir_util = dutils.dir_util
+log = dutils.log
```

Comment:
Suggestion: move the assignments below all import for flake8

Reply:
Fixed in 4c3948b.